Content.Server/GameObjects/Components/Items/ToysComponent.cs

```diff
@@ -24,36 +24,54 @@ public class ToysComponent : Component, IActivate, IUse, ILand
         [ViewVariables]
         [DataField("toySqueak")]
         public string _soundCollectionName = "ToySqueak";
+        private EmitSoundOnLandComponent? _emitSoundOnLand = default;
+        private EmitSoundOnUseComponent? _emitSoundOnUse = default;
```

Comment:
instead of doing this  ToysComponent should just be removed entirely, it just duplicates logic now--the premise of that issue was to replace ToysComponent with EmitSoundOnUse and EmitSoundOnLand

Reply:
ok, I'll redo this